### PR TITLE
rewording, removing passive voice

### PR DESCRIPTION
**Pull request by mihanizator**

### CODE REVIEW

File: config-1_x_0.md [hunk 1]
```diff
@@ -2,31 +2,26 @@
...
 
-There are two types of configurations. The first one is client-level config, which typically lives in your git
-repositories in the `Exofile.yml` file. Another config type is project-level config, which lives in the cloud.
+There are two types of configurations. The first one is client-level config, which typically lives in your git repositories in the `Exofile.yml` file. Another config type is project-level config, which lives in the cloud.
 
 There may be multiple client-level configs, but only one project-level config.
 
-Some handlers are allowed in the client-level config (Exofile.yml), only in the project-level config, or in both config
-types. Each handler documentation entry contains the information on which kind of config is supported.
+Some handlers are allowed in the client-level config (Exofile.yml), only in the project-level config, or in both config types. Each handler documentation entry contains the information on the config.
```
Comment (glebpom) [inline config-1_x_0.md]: Last sentence doesn't make sense

File: config-1_x_0.md [hunk 2]
```diff
@@ -142,7 +135,7 @@ rescue:
 
 ### Rescue
 
-Rescue blocks are used to catch exceptions and handle HTTP status codes generated by the particular handler.
+Rescue blocks help to catch exceptions and handle HTTP status codes generated by the particular handler.
```
Comment (glebpom) [inline config-1_x_0.md]: Not sure about using "help" here. It sounds like they optimize something, make it easier to implement. In fact, they are the only way made specifically to catch exceptions.

File: config-1_x_0.md [hunk 3]
```diff
@@ -607,11 +599,10 @@ Exogress edge servers may perform content optimization.
 ### WebP
 
 WebP is an image format employing both lossy and lossless compression, along with animation and alpha transparency.
-Developed by Google, it is designed to create smaller or better looking images compared to the JPEG, PNG, or GIF image
+Developed by Google, it was designed to create smaller or better-looking images than the JPEG, PNG, or GIF image
```
Comment (glebpom) [inline config-1_x_0.md:602]: Funny, it's a copy-paste from wikipedia :)

**Review by glebpom — CHANGES_REQUESTED** [review]

File: config-1_x_0.md (hunk 3 above)
Comment (mihanizator) [reply config-1_x_0.md:602]: actually, this one is a matter of choice. both options work well

**Review by glebpom — APPROVED** [review]

**Pull request merged**